TheMovieDb/Reviews/View/ReviewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ReviewCell.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 31/10/21.
+//
+
+import UIKit
+
+final class ReviewCell: UICollectionViewCell {
+    
+    // MARK: - Properties
+    static let reuseIdentifier =  String(describing: ReviewCell.self)
```

Comment:
you can move this into a protocol and use it as a extension 

```swift
final class ReviewCell: UICollectionViewCell, Reusable {
```

```swift 
protocol Reusable: class {
    static var reusableIdentifier: String { get }
}

extension Reusable {
    static var reusableIdentifier: String {
        return String(describing: self)
    }
}
```

Reply:
🤯  That is new for me. I need to learn all about protocol